packages/agoric-cli/lib/main.js

```diff
@@ -107,6 +107,11 @@ const main = async (progname, rawArgs, powers) => {
   program
     .command('deploy [script...]')
     .description('run a deployment script against the local Agoric VM')
+    .option(
+      '--allow-unsafe-plugins',
+      'CAREFUL: allow the deploy script full control over the Agoric VM',
```

Comment:
it's not just the VM, right? it's the entire Unix user account.

Reply:
So does `agoric deploy` as a whole (without the `--allow-unsafe-plugins` flag).  So I added a bigger top-level warning, too.